hypothesis-python/src/hypothesis/internal/compat.py

```diff
@@ -524,16 +524,16 @@ def b64decode(s):
 
 _cases = []  # type: list
```

Comment:
You can also delete `_cases` here - it was used to cache both positive and negative results, but the new version is more readable.